phpcs.xml.dist

```diff
@@ -148,6 +148,12 @@
     </properties>
   </rule>
 
+  <rule ref="SlevomatCodingStandard.Numbers.RequireNumericLiteralSeparator">
+      <properties>
+        <property name="minDigitsAfterDecimalPoint" value="5" />
```

Comment:
This doesn't seem to have an effect, does it?

`1_000` and `1_024` doesn't look nice. Does `minDigitsBeforeDecimalPoint: 2` help avoid that?

Reply:
By doing this, this rules does not apply to anything. See https://github.com/GoogleForCreators/web-stories-wp/commit/c1035b29431124aa19437b1e0dcb53aa563e2b0c. 

I am happy to leave it in, but makes the change a little pointless.

Reply:
Remove `minDigitsAfterDecimalPoint` also results in changes in the code.